css/form_subscribe.css

```diff
@@ -10,37 +10,37 @@
   padding-right: 2rem;
 }
 
-.form-title {
+.form_subscribe-title {
   font-weight: 300;
 }
 
-.form-wrapper {
+.form_subscribe__form--wrapper {
   border-block-end: 1px solid #FFFFFF;
   max-width: 250px;
   margin-bottom: 1rem;
 }
 
-.form-group {
+.form_subscribe__form--group {
   position: relative;
   padding: 0;
   border: none;
```

Comment:
Nice code, only one thing that I've noticed is, It seems that you rename the selector, but in your commit message is change selectors. Could you please have a look on that.